src/util/getQueryParams.js

```diff
@@ -1,7 +1,7 @@
 export default function getParameterByName(name, url) {
   if (!url) url = window.location.href;
   name = name.replace(/[\]]/g, "\\$&");
-  var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
+  var regex = new RegExp("[?&]" + name + "(=([^&#/]*)|&|#|$)"),
```

Comment:
Als ik het goed begrijp zorgt dit dat een / *na* de name=value werkt? Dus bijv /full/6?index=true/ ?

Reply:
Ja dit is correct